### PR TITLE
Display/admin page html

### PR DESCRIPTION
**Pull request by BrendanTurner1**
# Description
This will add the User and Overall buttons/views to the admin page.

# Contributors
@BrendanTurner1 

# Screenshot(s)
image
image
****

# Notes
Need to fill the views with data still

# Checklist
- [x] My PR has an appropriately descriptive and concise title.
- [x] My PR denotes any/all team members who contributed to it.
- [x] My code follows the Turing Style Guides and best practices.
- [x] I ran the code locally and verified that there are no visible errors.
- [x] feat: My PR clearly describes what feature I'm adding and any changes needed to make it work. _(if applicable)_

### CODE REVIEW

File: src/css/styles.css [hunk 1]
```diff
@@ -2,14 +2,14 @@ body {
   font-family: 'Jaldi', sans-serif;
   justify-content: space-evenly;
   background-color: #003249;
-  height: 100%;
+  height: 740px;
   width: 100%;
   margin: 0;
   overflow: hidden;
```
Comment (tenthwalker) [inline src/css/styles.css:8]: On my mac window standard view, this hidden setting gets overridden by the styling in the height setting, but without that set height it cuts off the datepicker.
Might be worth removing if the scroll isn't a total dealbreaker.

File: dist/index.html [hunk 2]
```diff
@@ -47,18 +47,66 @@
...
+        </div>
+        <div>
+          <p>1. Brendan 10oz</p>
+          <p>1. Brendan 10oz</p>
+          <p>1. Brendan 10oz</p>
+          <p>1. Brendan 10oz</p>
+          <p>1. Brendan 10oz</p>
+          <p>1. Brendan 10oz</p>
+          <p>1. Brendan 10oz</p>
+          <p>1. Brendan 10oz</p>
```
Comment (ericahagle) [inline dist/index.html:75]: Instead of using `<p>` tags, maybe we could use `<ol>` to get automatically numbered lists.

File: dist/index.html [hunk 3]
```diff
@@ -48,18 +48,47 @@
...
-    <section class="admin-view">
-      Buttons
-    </section>
-    <section class="user-data">
-      User Data
-    </section>
-    <section class="overall-data">
-      Overall Data
-    </section>
+    <aside>
```
Comment (tenthwalker) [inline dist/index.html:54]: add label for aria specificity

**Review by tenthwalker — APPROVED** [review]
looks great, we'll continue to style the admin view/add the dom updates in a separate PR.

**Pull request merged**